app/views/projects/_add_collaborator_modal.html.erb

```diff
@@ -1,3 +1,5 @@
+<script src="https://cdn.jsdelivr.net/bootstrap.tagsinput/0.8.0/bootstrap-tagsinput.min.js"></script>
```

Comment:
add this to packge.json instead

Reply:
@tachyons I'm not sure how to add this in package.json

Reply:
https://dev.to/mbackermann/how-and-when-to-use-yarn-on-rails-3jm4

Reply:
It seems like they don't provide any yarn version for this. Can you please confirm?

Reply:
Seems so https://github.com/bootstrap-tagsinput/bootstrap-tagsinput/issues/553. So you can put the flle in `vendor` dir

Reply:
On second thought it is not urgent, so merging this